gekitchen/erd/serializer.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from datetime import timedelta
+from textwrap import wrap
+from typing import Any, Dict, Optional, Set, Tuple, Union
+
+from .codes import ErdCode
+from .registry import _registry
+from .converters import ErdValueConverter
+from .converters.primitives import *
+
+_LOGGER = logging.getLogger(__name__)
+
+ErdCodeType = Union[ErdCode, str]
+
+class ErdSerializer:
```

Comment:
Is this used anywhere except https://github.com/ajmarks/gekitchen/pull/7/files#diff-f7dcdaf6a62f6adad7ea90ceca0165811c2efa8cc4a5ac395f5e0cc84ec0a204R27 ?

If not, since it doesn't have any instance or class variables, it should probably just be three functions without the class trappings.  They could either be directly folded into `GeAppliance`, left here with thin wrapper methods in `GeAppliance`, or just invoked directed when needed.

Reply:
In the fork, I restructured, and had thin wrappers in GeAppliance, so I think we ultimately got to the same page.